tests/checkLibraries.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+lambdapi='dune exec -- lambdapi check'
+TIMEFORMAT="%Es"
+out=/tmp/lambdapi.output
+
+checkoutLib() {
+    git clone $1 $2
+}
+
+checkLib() {
+    for f in $(find $1 -name *.lp)  $(find $1 -name *.dk)
+    do
+        echo lambdapi check $options $f ...
+        $lambdapi "$f" > $out 2>&1
+        if test $? -ne 0; then cat $out; exit 1; fi;
```

Comment:
I am wondering whether this cannot be simplified as follows:
```
set -e # to be added at the beginning of the file
[...]
do
  $lambdapi $options "$f"
done
```
The command `set -e` makes bash stops the script at the first error.
runtests.sh could be simplified accordingly
perhaps it has been written like this before the command `set -e` was added

Reply:
@fblanqui , I just noticed `ko_tests` is never called in `runtest.sh`. Is that on purpose? Should `ko_tests` be removed?

Reply:
Good remark. Yes, I think that it can be removed: ko tests are tests that should fail and they are tested in ok_ko.ml.

Reply:
Ok. I will remove ko_tests then.
For simplifying the call to lambdapi check, it seems the `if` structure is used to display the errors in case some check fails :
`if test $? -ne 0; then cat $out; exit 1; fi;;`
Are you sure we should remove it?

Reply:
I believe that it is not necessary anymore with `set -e` (if you do not redirect lambdapi output).